Formula/sqlite.rb

```diff
@@ -39,7 +38,7 @@ class Sqlite < Formula
 
   resource "docs" do
     url "https://sqlite.org/2016/sqlite-doc-3140200.zip"
```

Comment:
Doc URL needs to be updated too `https://sqlite.org/2016/sqlite-doc-3150000.zip`. Checksum is `cea0208b2b8dddf51b846d6e9d23210fc497da2c6f3fbed96f00eadceb1c32f7`.